src/runtime/CollectionWrappers/SequenceWrapper.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Collections.Generic;
+
+namespace Python.Runtime.CollectionWrappers
+{
+    internal class SequenceWrapper<T> : IterableWrapper<T>, ICollection<T>
+    {
+        public SequenceWrapper(PyObject pyObj) : base(pyObj)
+        {
+
+        }
+
+        public int Count
+        {
+            get
+            {
+                var size = Runtime.PySequence_Size(pyObject.Handle);
+                if (size == -1)
+                {
+                    Runtime.CheckExceptionOccurred();
+                    throw new Exception("Unable to get sequence size!");
```

Comment:
`CheckExceptionOccured` should throw in the line above

Reply:
@lostmsu according to python docs for PySequenceDelSlice:
Delete the slice in sequence object o from i1 to i2. Returns -1 on failure.

It doesn't explicitly say it will set an error on failure.  Is this implicit anywhere I see the word failure in the python doc?